src/pages/gen2/start/quickstart/vite-react-app/index.mdx

```diff
@@ -0,0 +1,270 @@
+export const meta = {
+  title: 'Vite + React App',
+  description: 'Get started with AWS Amplify (Gen 2) using React + Vite.'
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+This Quickstart guide will walk you through how to build a task list application with TypeScript, Vite and React. If you are new to these technologies, we recommend you go through the official [React](https://react.dev/learn/tutorial-tic-tac-toe), [Vite](https://vitejs.dev/guide/), and [TypeScript](https://www.typescriptlang.org/docs/handbook/typescript-from-scratch.html) tutorials first.
+
+import prerequisites from 'src/fragments/gen2/quickstart/prerequisites.mdx';
+
+<Fragments fragments={{ javascript: prerequisites, nextjs: prerequisites }} />
+
+## Create a project
+
+From your project directory, run the following command and respond to the prompts:
+
+```bash title="Terminal" showLineNumbers={false}
+npm create vite@latest
+✔ Project name: react-amplify-gen2
+✔ Select a framework: › React
+✔ Select a variant: › TypeScript
+```
+
+This creates a new React app in a directory called react-amplify-gen2. You can then run the following commands to switch into the new react-amplified directory and install the project dependencies:
+
+```bash title="Terminal" showLineNumbers={false}
+cd react-amplify-gen2
+npm install
+```
+
+## Create a backend
```

Comment:
@chrisbonifacio Curious why you're duplicating content here for this section instead of using the https://github.com/aws-amplify/docs/blob/main/src/fragments/gen2/quickstart/create-amplify.mdx fragment

Reply:
There's just some Next.js specific content in the fragment. Wasn't sure if there was a way to dynamically render content for the context of Vite instead using fragments?